BrainPortal/public/javascripts/cbrain.js

```diff
@@ -563,95 +498,44 @@
           complete: function() {
             loading_image.hide();
           }
-        }); 
-      }
-    });
-    
-    var filter_header_timeout = null; 
-    
-    $(document).delegate(".filter_header", "mouseenter", function() {
-      
-      if (filter_header_timeout) {
-        clearTimeout(filter_header_timeout);
-        filter_header_timeout = null;
+        });
       }
-      
-      var header = $(this);
-      var target = $(header.attr("data-target"));
-      var search = target.find(".filter_search");
-      
-      filter_header_timeout = setTimeout(function() {
-        target.show();
-        search.focus();
-      }, 500);
-      
-      header.closest("th").mouseleave(function() {
-        if (filter_header_timeout) {
-          clearTimeout(filter_header_timeout);
-          filter_header_timeout = null;
-        }
-        target.hide();
-        return false;
-      });
-      
-      return false;
     });
-    
-    $(document).delegate(".hover_open", "mouseenter", function() {
+
+
+    $(document).undelegate(".hover_open", "mouseenter").delegate(".hover_open", "mouseenter", function() {
       var header = $(this);
-      var target = $(header.attr("data-target"));
-      
+      var target = $(header.data("target"));
+
       target.show();
       header.mouseleave(function() {
```

Comment:
Find a way to somehow remove the handler before re-binding. This will add a new `mouseleave` event handler every time the target is hovered.